cmd/app/use.go

```diff
@@ -0,0 +1,108 @@
+package app
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/stormkit-io/stormkit-cli/api"
+	"github.com/stormkit-io/stormkit-cli/stormkit"
+	"github.com/spf13/cobra"
+)
+
+// useCmd represents the use command
+var useCmd = &cobra.Command{
+	Use:   "use",
+	Short: "Select an application to work on",
+	Long: `Select an application where to work on, via his repo name, id or index.
```

Comment:
**question**

Now that I think about this; the `index` is something that we can't really rely on. For instance, what happens if you choose the 3rd application and then delete the 2nd? Also, the `repository` name is not a unique value since a user can create multiple applications using the same repository. The only reliable value here seems to be application `id`. Should we use only that value? 

**suggestion**

If we're going to use the application id maybe we can refactor the `Long` string with something like:

```
Long: `Choose an application by providing the application id. The chosen application will be cached and will be used as the default application for subsequent commands.

Example: 

$ stormkit-cli app use --app-id <app_id>

Please note that this command will have no effect in case it is executed from a directory that contains a "stormkit.config.yml" file. The config file takes precedence over the "app use" command.
```

Reply:
That would work, at this point we can directly write the ID as argument (without the flag)

Like:

```
$ stormkit-cli app use <app_id>
```

WDYT?

Reply:
Waiting answer on [this](https://github.com/stormkit-io/stormkit-cli/pull/1#discussion_r516257357)

Reply:
Even better!

Reply:
Done in [ab3b64c](https://github.com/stormkit-io/stormkit-cli/pull/1/commits/ab3b64c669731d70eedf8149d62efcb5c1a714c0)